tests/security/oscap_ansible_anssi_bp_28_high/oscap_security_guide_setup.pm

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test 'pci-dss-4' hardening in the 'scap-security-guide' works: setup environment
+# Maintainer: QE Security <none@suse.de>
+# Tags: poo#93886, poo#104943
+
+use base 'oscap_tests';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils qw(is_sle);
+
+sub run {
+    my ($self) = @_;
+    $oscap_tests::sle_version = 'sle' . get_required_var('VERSION') =~ s/([0-9]+).*/$1/r;
+    $oscap_tests::evaluate_count = 3;
+    $oscap_tests::ansible_remediation = 1;
+    $oscap_tests::profile_ID = is_sle ? $oscap_tests::profile_ID_sle_anssi_bp28_high : $oscap_tests::profile_ID_tw;
```

Comment:
This should be a setting, we are repeating this test, for example:
`tests/security/oscap_ansible_stig/oscap_security_guide_setup.pm`
Same for the line 21, it could be parametrized and would save a lot of files.

Additionally I noticed you missed to select the console in some of these files more below.

Reply:
Fixed